formats/local_transaction/publisher/details.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "lgsl_code",
+    "lgil_override",
+    "service_tiers",
+    "introduction",
+    "more_information",
+    "need_to_know"
+  ],
+  "properties": {
+    "lgsl_code": {
+      "type": "integer"
+    },
+    "lgil_override": {
+      "anyOf": [
+        {
+          "type": "integer"
+        },
+        {
+          "type": "null"
+        }
+      ]
+    },
+    "service_tiers": {
```

Comment:
Could you include some descriptions for this field and `lgsl_code` and `lgil_override`?